dist/Dockerfile.deploy/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/app-sre/cincinnati:builder AS builder
 
+COPY . .
```

Comment:
We do not need to set the workdir ?

Reply:
Its already set in quay.io image and CI's replacement, no need to override it